src/services/disney.services.mjs

```diff
@@ -0,0 +1,60 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://api.disneyapi.dev",
+});
+
+const MAX_PAGE = 149;
+const MAX_ITEMS_PER_PAGE = 50;
+
+function randomPage(maxPage) {
+  return Math.floor(Math.random() * maxPage + 1);
+}
+
+function createCardObject(character) {
+  if (!character) throw new Error("Character is undefined");
+
+  const end = character.imageUrl.indexOf("/revision");
+
+  return {
+    id: character._id,
+    name: character.name,
+    image: end > -1 ? character.imageUrl.slice(0, end) : character.imageUrl,
+  };
+}
+
+async function getCards(amount) {
```

Comment:
Die getCards Funktionen ähneln sich alle sehr. Da könntest du sowas wie ne APIClientFactory draus machen.

Reply:
Habe ich in game.service.js als createCardObject gesetzt.

Reply:
In Utils-> getApiItemsFromMultiplePages